pvlib/clearsky.py

```diff
@@ -773,6 +809,22 @@ def detect_clearsky(measured, clearsky, times=None, window_length=10,
     sample_interval, samples_per_window = \
         tools._get_sample_intervals(times, window_length)
 
+    # if infer_limits, find threshold values using the sample interval
+    if infer_limits:
+        window_length, mean_diff, max_diff, lower_line_length, \
+            upper_line_length, var_diff, slope_dev = \
+                _clearsky_get_threshold(sample_interval)
+
+        # recalculate sample_interval, samples_per_window using returned
+        # window_length
+        sample_interval, samples_per_window = \
+            tools._get_sample_intervals(times, window_length)
```

Comment:
```suggestion
        # recalculate samples_per_window using returned
        # window_length
        _, samples_per_window = \
            tools._get_sample_intervals(times, window_length)
```
`sample_interval` shouldn't change, right?  (might be able to reflow this comment and code to fit on a single line now)

Reply:
The code line was still a little too long, but the comment does fit on one line now